hack/hypershift/package-template.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: selectorsyncset-template
+parameters:
+  - name: REGISTRY_IMG
+    required: true
+  - name: IMAGE_DIGEST
+    required: true
+objects:
+  - apiVersion: hive.openshift.io/v1
+    kind: SelectorSyncSet
+    metadata:
+      name: addon-operator-hypershift
+    spec:
+      clusterDeploymentSelector:
+        matchLabels:
+          ext-hypershift.openshift.io/cluster-type: service-cluster
```

Comment:
the Policy, PlacementRule and PlacementBinding objects go to the Service Cluster